drivers/ma_reports/app/models/ma_reports/monthly_performance/report.rb

```diff
@@ -334,14 +338,118 @@ def demographic_breakdowns
+    end
+
+    def race_col_lookup
+      {
+        'am_ind_ak_native' => 'AmIndAKNative',
+        'asian' => 'Asian',
+        'black_af_american' => 'BlackAfAmerican',
+        'native_hi_pacific' => 'NativeHIPacific',
+        'white' => 'White',
+        'hispanic_latinaeo' => 'HispanicLatinaeo',
+        'mid_east_n_african' => 'MidEastNAfrican',
+        'race_none' => 'RaceNone',
+      }
+    end
+
+    def race_columns
+      race_col_lookup.keys
+    end
+
+    private def pluck_to_hash(columns, scope)
+      scope.pluck(*columns.keys).map do |row|
+        Hash[columns.keys.zip(row)]
+      end
+    end
+
+    private def single_race(race_data, race, bucket_array)
+      bucket_array ||= []
+      race_data.select { |enrollment| enrollment[race] && !enrollment['hispanic_latinaeo'] }.
+        each do |e|
+          bucket_array << e['id'] if e.excluding('id', 'hispanic_latineao').count { |m| m.count(true) == 1 } == 1
+        end
+    end
+
+    private def single_race_latinaeo(race_data, race, bucket_array)
+      bucket_array ||= []
+      race_data.select { |enrollment| enrollment[race] && enrollment['hispanic_latinaeo'] }.
+        each do |e|
+          bucket_array << e['id'] if e.excluding('id', 'hispanic_latineao').count { |m| m.count(true) == 1 } == 1
+        end
+    end
+
+    private def lookups
+      {
+        am_ind_ak_native: ->(race_data, bucket) { single_race(race_data, 'am_ind_ak_native', bucket) },
+        am_ind_ak_native_hispanic_latinaeo: ->(race_data, bucket) { single_race_latinaeo(race_data, 'am_ind_ak_native', bucket) },
+        asian: ->(race_data, bucket) { single_race(race_data, 'asian', bucket) },
+        asian_hispanic_latinaeo: ->(race_data, bucket) { single_race_latinaeo(race_data, 'asian', bucket) },
+        black_af_american: ->(race_data, bucket) { single_race(race_data, 'black_af_american', bucket) },
+        black_af_american_hispanic_latinaeo: ->(race_data, bucket) { single_race_latinaeo(race_data, 'black_af_american', bucket) },
+        mid_east_n_african: ->(race_data, bucket) { single_race(race_data, 'mid_east_n_african', bucket) },
+        mid_east_n_african_hispanic_latinaeo: ->(race_data, bucket) { single_race_latinaeo(race_data, 'mid_east_n_african', bucket) },
+        native_hi_pacific: ->(race_data, bucket) { single_race(race_data, 'native_hi_pacific', bucket) },
+        native_hi_pacific_hispanic_latinaeo: ->(race_data, bucket) { single_race_latinaeo(race_data, 'native_hi_pacific', bucket) },
+        white: ->(race_data, bucket) { single_race(race_data, 'white', bucket) },
+        white_hispanic_latinaeo: ->(race_data, bucket) { single_race_latinaeo(race_data, 'white', bucket) },
+
+        hispanic_latinaeo: ->(race_data, bucket) do
+          bucket ||= []
+          race_data.select { |enrollment| enrollment['hispanic_latinaeo'] }.
+            each do |e|
+              bucket << e['id'] if e.excluding('id').count { |m| m.count(true) == 1 } == 1
+            end
```

Comment:
Let's make a method `only_latinaeo` that mirrors the behavior of `single_race` but is specific to hispanic_latinaeo[

(Same with Multi-racial and multi-with-hispanic/latin@.)

That should allow `def lookups` to be identical for this file and `drivers/system_pathways/app/models/system_pathways/equity/race_and_ethnicity.rb`.

Different methods will be defined for `only_latinaeo` etc.